upgrade/upgrade-1.1.2.php

```diff
@@ -1,31 +1,44 @@
 <?php
 
-function upgrade_module_1_1_2(Module $module)
+use Gett\MyparcelBE\Database\Table;
+
+/**
+ * @throws \PrestaShopDatabaseException
+ */
+function upgrade_module_1_1_2(MyParcelBE $module): bool
 {
-    foreach(Db::getInstance()->executeS('
-        SELECT `carrier`.* FROM `ps_carrier` AS `carrier`
-	    LEFT JOIN `ps_myparcelbe_carrier_configuration` AS `config` ON `carrier`.`id_carrier` = `config`.`id_carrier` AND `config`.`name` = "carrierType"
-	    WHERE `carrier`.`external_module_name` = "' . $module::MODULE_NAME . '"
-	    AND `config`.`id_configuration` IS NULL;
-    ') as $record) {
+    $carrier              = Table::withPrefix('carrier');
+    $carrierConfiguration = Table::withPrefix(Table::TABLE_CARRIER_CONFIGURATION);
+
+    $query = <<<SQL
+SELECT carrier.* FROM $carrier AS carrier
+  LEFT JOIN $carrierConfiguration 
+    AS config 
+    ON carrier.id_carrier = config.id_carrier AND config.name = carriertype
+WHERE carrier.external_module_name = {$module::MODULE_NAME}
+AND config.id_configuration IS NULL
+SQL;
+
+    foreach (Db::getInstance()
+                 ->executeS($query) as $record) {
         if (preg_match('/Post\s?NL/', $record)) {
             $carrierType = \Gett\MyparcelBE\Constant::POSTNL_CARRIER_NAME;
-        } elseif (preg_match('/DPD/', $record)) {
+        } elseif (strpos($record, 'DPD')) {
             $carrierType = \Gett\MyparcelBE\Constant::DPD_CARRIER_NAME;
         } else {
             $carrierType = $module->isNL()
                 ? \Gett\MyparcelBE\Constant::POSTNL_CARRIER_NAME
-                : \Gett\MyparcelBE\Constant::BPOST_CARRIER_NAME
-            ;
+                : \Gett\MyparcelBE\Constant::BPOST_CARRIER_NAME;
```

Comment:
Persoonlijk zou ik CONSTANTS ook in een use statement zetten.

Reply:
Ik durf dat eigenlijk niet, maar yolo